train.py

```diff
@@ -0,0 +1,153 @@
+import hydra
+import torch
+import numpy as np
+from tqdm import tqdm
+from torch import nn, optim
+from torch.utils.data import DataLoader
+from omegaconf import OmegaConf, DictConfig
+
+import wandb
+from model.VQVAE import VQVAE
+from utils.data_loader import create_loaders
+
+
+def count_parameters(model):
+    # https://discuss.pytorch.org/t/how-do-i-check-the-number-of-parameters-of-a-model/4325/9
+    print(f"The model has {sum(p.numel() for p in model.parameters() if p.requires_grad):,} trainable parameters.")
+
+
+def set_seed(seed):
+    # https://pytorch.org/docs/stable/notes/randomness.html
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed_all(seed)
+    np.random.seed(seed)
+    torch.backends.cudnn.deterministic = True
+    torch.backends.cudnn.benchmark = False
+
+
+def test(model: VQVAE, test_loader: DataLoader, criterion, cfg: DictConfig, epoch: int):
```

Comment:
```suggestion
def test_step(model: VQVAE, test_loader: DataLoader, criterion, cfg: DictConfig, epoch: int):
```

Same as with `train_step`